website/docs/r/cm_subnet.markdown

```diff
@@ -0,0 +1,60 @@
+---
+layout: "rightscale"
+page_title: "Rightscale: subnet"
+sidebar_current: "docs-rightscale-resource-subnet"
+description: |-
+  Create and maintain a rightscale cloud management subnet.
+---
+
+# rightscale_subnet
+
+Use this resource to create, update or destroy rightscale [subnets](http://reference.rightscale.com/api1.5/resources/ResourceSubnets.html).
```

Comment:
RightScale

Reply:
Will fix up in big cleanup before 1.0 release.